.travis.yml

```diff
@@ -0,0 +1,62 @@
+os: linux
+dist: xenial
+sudo: false
+
+language: c
+
+addons:
+  apt:
+    packages:
+     - build-essential
+     - perl
+     - curl
+
+compiler:
+  - gcc
+  - clang
+
+env:
+  global:
+    - JOBS=2
+    - WORK=$TRAVIS_BUILD_DIR/work
+    - PREFIX=$HOME/kopenresty
+    - LUAROCKS=3.1.3
```

Comment:
should `LUAROCKS` be a `matrix` option instead?

Reply:
Don't think it's worth it. It'd significantly increase the build time, it isn't tightly coupled (in terms of dependency requirements) with either of OpenResty/OpenSSL, and is already part of CI in other repositories, including Kong/kong.